src/platforms/hubspot/actions/associations/create.ts

```diff
@@ -0,0 +1,57 @@
+import { action } from '@/sdk';
+import client from '../../client';
+import { z } from 'zod';
+
+import { transformAssociation } from '@/platforms/hubspot/actions/mappers';
+import { HUBSPOT_DEFINED_CATEGORY, hubspotModuleSchema } from '../../schemas';
+
+export default action(
+  'associations-create',
+  {
+    operation: 'create',
+    resource: 'associations',
+    mutation: true,
+    schema: z.object({
+      fromId: z.string(),
+      fromType: hubspotModuleSchema,
+      toId: z.string(),
+      toType: hubspotModuleSchema,
+      category: z.string().optional(),
+      typeId: z.string().optional(),
+    }),
+    scopes: [
+      'crm.schemas.companies.write',
+      'crm.schemas.contacts.write',
+      'crm.schemas.deals.write',
+    ],
+  },
+  async ({ auth, input }) => {
+    const { category, typeId } = await (async () => {
```

Comment:
a little confused about what this is doing, what's the category and typeId for?

Reply:
Basically copying from here:
https://github.com/vesselapi/monorepo/blob/84f8400689e15d88e08bd2753e75d914eb46fd3a/packages/common/integrations/integrations/connector/crm/hubspotConnector/HubspotApiHandler.ts#LL391C1-L391C1